app/src/main/res/layout/dialog_markdown_view.xml

```diff
@@ -1,16 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:paddingLeft="16dp"
-    android:paddingRight="16dp">
+    android:layout_height="match_parent">
 
     <TextView
         android:id="@+id/markdown_view"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginTop="@dimen/default_item_margin"
-        android:textIsSelectable="false"
-        android:scrollbars="vertical" />
+        android:textIsSelectable="true"
```

Comment:
If I'm not wrong I need to disable the `textIsSelectable` attribute to be able to open URL links.

Reply:
Confirmed that the links are no longer clickable.

Reply:
Done